frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/configs/InboxConfig.js

```diff
@@ -37,8 +37,81 @@ const inboxConfig = (t) => {
             },
             searchResult : {
                 label : "",
-                uiConfig : {
-                    
+                uiConfig: {
+                    defaultValues: {
+                        offset: 0,
+                        limit: 10,
+                        // sortBy: "department",
+                        sortOrder: "ASC",
+                    },
+                    columns: [
+                        {
+                            label: "name",
+                            jsonPath: "searchResult[0].name",
+                            accessor: "name",
+                            isLink: true,
+                            redirectUrl: '/works-ui/employee/project/project-inbox-item'
+                        },
+                        {
+                            label: "age",
+                            jsonPath: "searchResult[0].age",
+                            accessor: "age",
+                            isLink: false,
+                            redirectUrl: ""
+                        },
+                        {
+                            label: "gender",
+                            jsonPath: "searchResult[0].age",
+                            accessor: "gender",
+                            isLink: false,
+                            redirectUrl: ""
```

Comment:
keep redirectURL as optional, add only if redirection is required.
else don't add